.github/actions/build-ruby/action.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Build Ruby'
+description: 'Builds the given Ruby version'
+author: New Relic
+inputs:
+  ruby-version:
+    description: 'Ruby version'
+    required: true
+    default: '2.0.0-p648'
```

Comment:
We agreed we'll probably want to change this to pick up the latest version of Ruby so we can do periodic test runs to see whether we pass against any recent releases.